build_tools/android/common/benchmark_description.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python3
```

Comment:
It's intended for various steps of the whole pipeline so that they can have a consistent view of the benchmarks. `run_benchmarks.py` generates benchmark results according to the data structure in `common`, and later down the pipeline we need to "decode" the benchmark results and push to dashboard or pull request.